Teak/TeakSession.m

```diff
@@ -277,7 +277,7 @@ - (void)sendUserIdentifier {
                                                       NSString* deepLink = reply[@"deep_link"];
                                                       NSURL* url = [NSURL URLWithString:deepLink];
                                                       if (url && blockSelf.launchDataOperation != nil) {
-                                                        [blockSelf.launchDataOperation.result updateDeepLink:url];
+                                                        blockSelf.launchDataOperation = [blockSelf.launchDataOperation updateDeepLink:url];
```

Comment:
This will suppress the original short URL, whereas we preserve it on Android.

Reply:
Yep. There were some bigger changes needed to preserve it, and so I decided I'd do it in a future update.

Reply:
Changes made.